From f067527fd6fd97f349a7e1ffb8df0bbc4899058b Mon Sep 17 00:00:00 2001
From: Ciprian Costea <ciprianmarian.costea@nxp.com>
Date: Fri, 13 May 2022 10:43:12 +0300
Subject: [PATCH 01/30] dt-bindings: Add definitions for reserved ddr errata
 memory region

In order to sync dtb with A-TF and Linux.

Issue: ALB-8823
Issue: ALB-8781
Upstream-Status: Pending 

Signed-off-by: Ciprian Costea <ciprianmarian.costea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/dt-bindings/ddr-errata/s32-ddr-errata.h | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 include/dt-bindings/ddr-errata/s32-ddr-errata.h

diff --git a/include/dt-bindings/ddr-errata/s32-ddr-errata.h b/include/dt-bindings/ddr-errata/s32-ddr-errata.h
new file mode 100644
index 0000000000..f37db8da24
--- /dev/null
+++ b/include/dt-bindings/ddr-errata/s32-ddr-errata.h
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright 2020-2022 NXP
+ */
+
+#ifndef __DT_BINDINGS_S32_DDR_ERRATA_H__
+#define __DT_BINDINGS_S32_DDR_ERRATA_H__
+
+#define DDR_ERRATA_REGION_BASE 0x85000000
+#define DDR_ERRATA_REGION_SIZE 0x1000
+
+#endif
-- 
2.17.1

